app/src/main/java/com/islaharper/dawnofandroid/presentation/screens/splash/SplashScreen.kt

```diff
@@ -48,24 +47,32 @@ fun SplashScreen(
             }
         }
     }
-    Splash(composition, progress)
+    if (composition != null) {
```

Comment:
In what circumstances will the `LottieComposition` be `null`?

Reply:
From the Lottie Repo https://github.com/airbnb/lottie/blob/master/android-compose.md:

"This will return null until the composition is parsed and then will return the LottieComposition object."